### ISSUE TITLE
Android NavigationHeader not receiving press

### ISSUE THREAD

**Issue by cdanwards** [ApollosProject/apollos-prototype#695]
### Details

Currently on Android, when navigating to a `ContentSingle` in the Navigation Header, the `X` close button is not responding to press. Currently I'm unable to exit a content single without using the android native back button.

### Expected Behavior

When I press the close `X` button, that it responds and closes the view.

### Actual Behavior

Currently, the button does not respond.

### Possible Fix

@conrad-vanl mentioned it could be an issue with a more recent version of React Native.

<details><summary>Additional Info</summary>

### Your Environment

- Environment name and version (e.g. Chrome 39, node.js 5.4): Pixel 2 XL
- Operating System and version (desktop or mobile): Android SDK 28

### Steps to Reproduce

![Android-UI](https://user-images.githubusercontent.com/8878532/57406141-1e69a900-71ae-11e9-82e4-5eeffc0de21a.gif)

### Stack Trace

</details>

**Comment by cdanwards** [comment]
Just some notes: 
The issue stems from the use of `position: absolute` [here](https://github.com/ApollosProject/apollos-prototype/blob/523ea1c319ef5eb06038ee373a63fb4f0acb0d4b/packages/apollos-ui-kit/src/ModalView/ModalViewHeader.js#L17).
I've looked into whether or not the underlying `TouchableNativeFeeback` has issues in the `Touchable` of the `ButtonIcon` and looked into the `ButtonIcon` but I've narrowed it down to that styling. The thing is, we're following the recommended pattern of wrapping the `ButtonIcon`(touchable component) within a `View` that has the positioning on it.
